assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -437,8 +437,39 @@ describe("DiversionPage", () => {
       />
     )
 
-    expect(container.querySelectorAll(".c-stop-icon-container")).toHaveLength(
-      11
+    expect(container.querySelectorAll(".c-stop-icon")).toHaveLength(11)
+  })
+
+  test("missed stop markers are drawn on the map", async () => {
+    const stop1 = stopFactory.build()
+    const stop2 = stopFactory.build()
+    const stop3 = stopFactory.build()
+    const stop4 = stopFactory.build()
+
+    jest
+      .mocked(fetchFinishedDetour)
+      .mockResolvedValue(finishedDetourFactory.build({ missedStops: [stop2] }))
+
+    const { container } = render(
+      <DiversionPage
+        originalRoute={{
+          shape: shapeFactory.build({ stops: [stop1, stop2, stop3, stop4] }),
+        }}
+      />
     )
+
+    await act(async () => {
+      fireEvent.click(originalRouteShape.get(container))
+    })
+    await act(async () => {
+      fireEvent.click(originalRouteShape.get(container))
+    })
+
```

Comment:
question: do these need to be async?
```suggestion
    await act(() => {
      fireEvent.click(originalRouteShape.get(container))
    })
    await act(() => {
      fireEvent.click(originalRouteShape.get(container))
    })

```

Reply:
[They do not](https://github.com/mbta/skate/pull/2484/commits/1e65828bb35b13d0fbccc3ac9a5d0217b9ec45a4)